src/collection/views/CollectionEdit.tsx

```diff
@@ -115,7 +118,7 @@ const CollectionEdit: FunctionComponent<CollectionEditProps> = ({ history, match
 		if (!tempCollection) {
 			toastService(
 				'De collectie updaten is mislukt, kon geen kopie maken van de bestaande collectie',
-				TOAST_TYPE.DANGER
+				DANGER
```

Comment:
dit zal niet werken met de nieuwste code
gebruik in de plaats daarvan:
```
toastService.danger()
```

overal elders aanpassen en laatste versie van release/19 nog eens mergen